tensorboard/webapp/runs/views/runs_table/runs_table_test.ts

```diff
@@ -27,6 +27,7 @@ import {
   TestBed,
 } from '@angular/core/testing';
 import {MatCheckboxModule} from '@angular/material/checkbox';
+import {MatDialogModule} from '@angular/material/dialog';
```

Comment:
Make sure BUILD file has dependency on expect_mat_dialog.

Reply:
yes it does have  `"//tensorboard/webapp/angular:expect_angular_material_dialog",`